api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -63,6 +72,19 @@
 	        OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_PATIENT_IDENTIFIER_VALIDATOR,
 	        OpenmrsConstants.LUHN_IDENTIFIER_VALIDATOR);
 	
+	private ConceptService cs;
+	private final String separator = FileSystems.getDefault().getSeparator();
+	
+	@Before
+	public void setup() {
+		cs=Context.getConceptService();
+	}
+	private File createFileInUserDirectory(String relativePath) {
+		String userDirectoryPath = System.getProperty("user.dir");
```

Comment:
For running testcases, are you able to use this method to make a directory in the ${user} directory `OpenmrsUtil.getDirectoryInApplicationDataDirectory()` ?

Reply:
It works in a way that when running tests, files are created in temp dummy files and when running the webapplication , the files are created in the expected directory.

Reply:
When im doing that createnewfile() throw expection

Reply:
I can't use this method. It throw exception when I want to create file.

Reply:
I believe that method was created for that! You should tweak around code and make it work!